Argo/Types/JSON.swift

```diff
@@ -13,14 +13,7 @@ public extension JSON {
   static func parse(json: AnyObject) -> JSON {
     switch json {
     case let v as [AnyObject]: return .Array(v.map(parse))
-
-    case let v as [Swift.String: AnyObject]:
-      return .Object(reduce(v, [:]) { (var accum, elem) in
-        let parsedValue = (self.parse <^> elem.1) ?? .Null
-        accum[elem.0] = parsedValue
-        return accum
-      })
-
+    case let v as [Swift.String: AnyObject]: return .Object(parse <^> v)
```

Comment:
Although Runes doesn't define a map operator for `Dictionary`, removing the import at this point causes a build error with the message `Operator is not a known binary operator`.